lms/models/region.py

```diff
@@ -1,38 +1,25 @@
 from dataclasses import dataclass
 
 
-@dataclass(frozen=True)
+@dataclass
 class Region:
     """Codes and details related to a deployment region."""
 
     code: str
     """Region code (e.g. ISO 3166 alpha-2 country code)."""
 
-    name: str
-    """Human readable description of the region."""
```

Comment:
I think moving `name` into `__post_init__()` also means it'll no longer be considered by the autogenerated `__repr__()` and `__eq__()`, but I don't think that matters.